### PR TITLE
FixCommand - Output warning message when both config and rules options are passed

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Probably best to target this at 2.1 since it's a feature?

**Comment by keradus** [comment]
please focus on code rather the target ;) (for most cases) I could easily change the target during merging.

File: src/Console/Command/FixCommand.php [hunk 1]
```diff
@@ -141,20 +144,26 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
-        }
+        if (null !== $stdErr) {
+            if (extension_loaded('xdebug')) {
+                $stdErr->writeln(sprintf($stdErr->isDecorated() ? '<bg=yellow;fg=black;>%s</>' : '%s', 'You are running php-cs-fixer with xdebug enabled. This has a major impact on runtime performance.'));
+            }
 
-        $configFile = $resolver->getConfigFile();
+            if (null !== $passedConfig && null !== $passedRules) {
+                $stdErr->writeln(array(
+                    sprintf($stdErr->isDecorated() ? '<bg=yellow;fg=black;>%s</>' : '%s', 'When passing both "--config" and "--rules" the configuration file not to be used.'),
```
Comment (keradus) [inline src/Console/Command/FixCommand.php]: config file is used, but not ruleset defined in it. (eg, finder will be used)

File: src/Console/Command/FixCommand.php [hunk 2]
```diff
@@ -141,20 +144,26 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+                    sprintf($stdErr->isDecorated() ? '<bg=yellow;fg=black;>%s</>' : '%s', 'Passing both options is deprecated; version v3.0 PHP-CS-Fixer will exit with an configuration exit code.'),
```
Comment (keradus) [inline src/Console/Command/FixCommand.php]:
```diff
-configuration exit code
+configuration error exit code
```
?

**Comment by keradus** [comment]
:+1: but minors

**Comment by keradus** [comment]
I see no feature here, just better DX with more verbose CLI self-documentation comments

**Comment by keradus** [comment]
@SpacePossum , would you mind adding the same warning for path when not in intersect mode ?

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request closed without merging**